zipline/utils/test_utils.py

```diff
@@ -881,3 +881,9 @@ def decorator(f):
         param_sets = product(*(params[name] for name in argnames))
         return subtest(param_sets, *argnames)(f)
     return decorator
+
+
+def make_test_handler(testcase, *args, **kwargs):
```

Comment:
Sure. I just moved this here as is from qexec's `test_utils` (with the intention of getting rid of it there, since qexec depends on zipline). Of course, if we don't end up using it to log missing sids, probably doesn't need to be here.